frontend/lib/models/execution_metadata.dart

```diff
@@ -78,10 +82,7 @@ abstract class ExecutionMetadata with _$ExecutionMetadata {
 
   List<String> toQueryParams() {
     return toRows()
-        .map(
-          (row) =>
-              '${Uri.encodeComponent(row.$1)}:${Uri.encodeComponent(row.$2)}',
-        )
+        .map((row) => base64Encode(utf8.encode('${row.$1}:${row.$2}')))
```

Comment:
You must encode/decode category and value separately. Otherwise with category `charm_qa:charm` and value `kratos`, you get `charm_qa:charm:kratos`, but when you split it it turns into category `charm_qa` and value `charm:kratos`.